config/initializers/google_cloud_trace.rb

```diff
@@ -1,15 +1,5 @@
 # frozen_string_literal: true
 
-# Fake google cloud trace in_span method so we don't have to check our environment
-# where we use it
-unless Rails.env.production?
-  module Google
-    module Cloud
-      class Trace
-        def self.in_span(_span_name)
-          yield
-        end
-      end
-    end
-  end
-end
+require "google/cloud/trace/rails"
```

Comment:
https://github.com/googleapis/google-cloud-ruby/blob/1a7cfef70298276a9203658748e9d70edfcfb2d5/google-cloud-trace/lib/google/cloud/trace/rails.rb#L38

Reply:
Where does `Cloud.configure.use_trace` get set?

Reply:
It only gets set to `true` if credentials are valid and if it's Production: https://github.com/googleapis/google-cloud-ruby/blob/1a7cfef70298276a9203658748e9d70edfcfb2d5/google-cloud-trace/lib/google/cloud/trace/rails.rb#L133-L142